bazel_tools/proto.bzl

```diff
@@ -159,3 +165,95 @@ proto_gen = rule(
 
 def _is_windows(ctx):
     return ctx.configuration.host_path_separator == ";"
+
+def maven_tags(group, artifact_prefix, artifact_suffix):
+    if group and artifact_prefix and artifact_suffix:
+        return ["maven_coordinates=%s:%s-%s:__VERSION__" % (group, artifact_prefix, artifact_suffix)]
+    else:
+        return []
+
+def proto_jars(
+        name,
+        srcs,
+        strip_import_prefix = "",
+        deps = [],
+        proto_deps = [],
+        java_deps = [],
+        scala_deps = [],
+        file_root = None,
+        javadoc_root_packages = [],
+        maven_group = None,
```

Comment:
It feels a bit weird, that this is the one thing where you specify those separately rather than via the tag.

Reply:
I know, but it's just more flexible this way. Concatenation seems safer than splitting.

Reply:
Currently, `proto_jars` hides the `tags` attribute. It's usually better to expose it, as there are a couple meaningful tags that developers may want to set. However, once `tags` is exposed this will need to guard against maven coords that are provided via `tags` instead of the `maven_*` attributes.

Reply:
I'll let whoever needs it add that. :stuck_out_tongue: 

(Let's be honest, it's going to be me, but at least it'll be _future_ me.)